docs/workflows/index.rst

```diff
@@ -22,7 +22,12 @@ accordingly. If you prefer to specify the username and password with each reques
 To make these workflows copy/pastable, we make use of environment variables. The first variable to
 set is the hostname and port::
 
-   $ export BASE_ADDR=http://<hostname>:24817
+   export BASE_ADDR=http://<hostname>:24817
+
+
+Furthermore, it is recommended to install the ``ostree`` utility. The utility is used to demonstrate
```

Comment:
What is the ostree utility? What is a "utility"?  Where can I find out more about it?

Reply:
If I read on, it appears to be a package manager? Utility is such a general word, can we use something more specific and can we give an immediate example of what this looks like?

Reply:
We can refer to it as a package manager in Pulp - https://manpages.debian.org/testing/ostree/ostree.1.en.html. I am adding a link to the manpage and will reword the statement.